src/imago.js

```diff
@@ -36,8 +41,8 @@
       e.preventDefault();
       e.stopPropagation();
 
-      if (e.which !== 1)
-        return;
+      // if (e.which !== 1)
+      //   return;
```

Comment:
this condition disable right mouse button, do you commented it with which reason?